README.md

```diff
@@ -278,6 +278,7 @@ Set up your GitHub Actions workflow with a specific version of your programming
 - [Secret Spreader](https://github.com/webfactory/secret-spreader) - Not an action per se, but a tool to manage Actions Secrets across a list of repositories.
 - [Secrets Sync Action](https://github.com/google/secrets-sync-action) - Action syncs secrets across multiple repositories.
 - [Snyk Test Action](https://github.com/snyk/actions)
+- [Manage Your GitHub Actions secrets, With A Simple CLI](https://github.com/unfor19/githubsecrets)
```

Comment:
Please follow title casing requirements found here: https://github.com/sdras/awesome-actions/blob/master/contributing.md. Also, you do not need the comma.

Reply:
@jessicalostinspace Thank you for your input, updated